releasenotes/notes/vqe-factory-improvements-02f411513f11907c.yaml

```diff
@@ -0,0 +1,30 @@
+---
+deprecations:
+  - |
+    :class:`~qiskit_nature.algorithms.VQEUCCFactory`
+    had setter/getter properties that were repeated with respect to
+    :class:`qiskit.algorithms.VQE`.
+    
+    In order to avoid duplicating all of these attributes, potentially leading to inconsistencies
+    between the attributes of the different classes, these are now deprecated and require you to use
+    the attributes from
+    :attr:`~qiskit_nature.algorithms.VQEUCCFactory.minimum_eigensolver`
+    instead.
+    
+    For this reason, the constructor of
+    :class:`~qiskit_nature.algorithms.VQEUCCFactory` has been changed as well.
+    Now the constructor only has the following positional arguments:
+    * initial_point
```

Comment:
```suggestion

    * initial_point
```
Sphinx does not treat this as a bullet list and needs a blank line to avoid wrapping the text into the paragraph
![image](https://user-images.githubusercontent.com/40241007/173128151-def11372-bc74-48df-a552-ee034b080d0b.png)

I will also note, that references to VQE etc, in another repo/set of docs, get high highlighted, but since its not all built together the do not end up as links. I think this is ok, not sure if you were aware of that.